src/attr/_make.py

```diff
@@ -366,7 +366,11 @@ def _is_class_var(annot):
     annotations which would put attrs-based classes at a performance
     disadvantage compared to plain old classes.
     """
-    return str(annot).startswith(_classvar_prefixes)
+    annot = str(annot)
```

Comment:
I'd prefer something safer here like:
```
    if annotation.startswith(("'", '"')) and annotation.endswith(("'", '"')):
        annotation = annotation[1:-1]
```
To handle the "double quoted" annotations.

Reply:
Agreed. I think I was just tired/frustrated.